src/components/Badge/Indicator/Indicator.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import cx from "classnames";
+import styles from "./Indicator.module.scss";
+import { IndicatorColor } from "./IndicatorConstants";
+import { ComponentDefaultTestId } from "../../../tests/constants";
+
+export interface IndicatorProps {
+  color?: IndicatorColor;
+}
+
+const Indicator: React.FC<IndicatorProps> & { colors?: typeof IndicatorColor } = ({
+  color = IndicatorColor.NOTIFICATION
+}: IndicatorProps) => {
+  return <div className={cx(styles.indicator, styles[color])} data-testid={ComponentDefaultTestId.INDICATOR} />;
```

Comment:
use our regular function for calculate data test id or not set data test id at all :)

Reply:
Since I removed VibeComponentProps (as the component is not "exported") there's no id, so I didn't see a reason for using the getTestId
But on second thought, there's no reason not allowing passing VibeComponentProps to the element